CONTRIBUTING.md

```diff
@@ -283,3 +283,11 @@ If you have your SSH keys set up on the staging environment (and can connect thr
 ### Adding SSH keys
 
 Add your SSH keys to `./staging/configuration.nix` and let existing owners deploy them.
+
+## Operators guidance
+
+### Using a Sentry-like collector
+
+You can set `GLITCHTIP_DSN` as a credential secret with a DSN and this will connect to a Sentry-like endpoint via your DSN.
```

Comment:
What's a DSN, can you please link to a definition? What's Sentry? These names are so generic that we need concrete definitions.

Reply:
OK, I will put links.